src/components/tetris/Tetris.module.scss

```diff
@@ -86,6 +86,52 @@
   }
 }
 
+@media (any-hover: none) {
+  .Tetris .buttons {
+    display: grid;
+    grid-template-columns: 1fr 2fr 1fr;
+    grid-gap: 1rem;
+    position: relative;
+    z-index: 9999;
+    margin-top: 1rem;
+    margin-right: 4rem;
+    margin-left: -2.2rem;
+    color: white;
+
+    > button {
+      font-size: 3rem;
+      pointer-events: all;
+      height: 100%;
+    }
+
+    button {
+      background: #2acff5;
+      color: white;
+    }
+  }
+}
+
+.alignTop {
+  display: grid;
+  justify-content: center;
+  grid-auto-flow: column;
+  padding: 3rem;
+  column-gap: 2.5rem;
+  margin-right: 2.5rem;
+}
```

Comment:
This was also moved to the top of the file in current main, to easier match the rendering order from top to bottom of various elements